### DISCUSSION TITLE
List index out of range error

### DISCUSSION THREAD

**Discussion by comejar** [svg2mod/svg2mod#59] [Q&A]
I get the following error

Also the online version of this isn't working, even with the example svg file so I'm not sure what is going wrong

Is this project not working and dead or is it something wrong on my end?

Parsing SVG...
Found SVG layer: Drill.Mech
Found SVG layer: Edge.Cuts
Found SVG layer: F.Cu
Found SVG layer: F.Mask
Found SVG layer: F.SilkS
Writing module file: poop.kicad_mod
←[91m←[7mUnhandled exception (Exiting)
 IndexError: list index out of range ←[0m

**Comment by Sodium-Hydrogen** [comment]
I can't help much with the web version as it is ran by someone else.

I would be happy to help with any issues you might encounter with the python package though. However, I need more information. Would you please fill out a [bug report]?


[bug report]:https://github.com/svg2mod/svg2mod/issues/new?assignees=&labels=bug%2Cunconfirmed&projects=&template=bug_report.yml
